operations/enterprises.md

```diff
@@ -309,8 +316,11 @@ Returns all spaces of an enterprise associated with the connector integration.
 | `Id` | string | required | Unique identifier of the category. |
 | `IsActive` | bool | required | Whether the space category is still active. |
 | `Name` | string | required | Name of the category. |
+| `Names` | object | required | All existing translations of the name accessible by [Language code](configuration.md#language). |
 | `ShortName` | string | optional | Short name \(e.g. code\) of the category. |
+| `ShortNames` | object | optional | All existing translations of the short name accessible by [Language code](configuration.md#language). |
```

Comment:
it is always required. And instead of object, whose structure you need to always describe, define an object which would be linked from here.